cnd/src/http_api/halight_herc20/alice.rs

```diff
@@ -121,7 +121,7 @@ impl FundAction for AliceSwap<asset::Bitcoin, asset::Erc20, halight::Finalized,
                 },
                 beta_finalized:
                     herc20::Finalized {
-                        state: herc20::State::Funded { .. },
+                        state: herc20::State::None,
```

Comment:
:exclamation: How did the e2e tests ever pass?

Reply:
> Could we somehow assert the funding on the Lightning side for Alice in this scenario?

What do you mean? I have corrected that already in 2b3183144ed3a24f29e1232b422026e5813c6810 and 016b1dc4c8d371553ea1bf27ad7312ea4c7b2f2a.